extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/OrdinalData.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.kubernetes.client.runtime;
+
+final class OrdinalData {
```

Comment:
I'd make this an `enum`, but that's just me.

Reply:
Good point, I'll fix it